tests/apps/vadd_async_mmap/run_tapa.bats

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2024 RapidStream Design Automation, Inc. and contributors.
+# All rights reserved. The contributor(s) of this file has/have agreed to the
+# RapidStream Contributor License Agreement.
+
+compile_host() {
+  cd "${BATS_TEST_DIRNAME}"
+
+  tapa g++ \
+    vadd-host.cpp vadd.cpp \
+    -o ${BATS_TMPDIR}/vadd-host
+
+  [ -x "${BATS_TMPDIR}/vadd-host" ]
+}
+
+compile_xo() {
+  cd "${BATS_TEST_DIRNAME}"
+
+  ${RAPIDSTREAM_TAPA_HOME}/usr/bin/tapa \
+    -w ${BATS_TMPDIR}/vadd-workdir \
```

Comment:
please change all occurrence of `vadd` to `async-mmap-vadd` as this will conflict with `vadd` if both tests are executed in parallel.